gtfs/tests/test_routes_api.py

```diff
@@ -94,3 +103,45 @@ def test_route_ordering(maas_api_client):
     response = maas_api_client.get(ENDPOINT)
     assert response.status_code == 200
     assert [r["name"] for r in response.data] == ["first", "second", "third"]
+
+
+@pytest.mark.django_db
+def test_rider_categories_with_prices(
+    maas_api_client, snapshot, api_id_generator, django_assert_max_num_queries
+):
+    feed = get_feed_for_maas_operator(maas_api_client.maas_operator, True)
+    route = baker.make(Route, feed=feed)
+    fare = baker.make(
+        Fare,
+        feed=feed,
+        source_id=seq("source_id of test fare "),
+        api_id=api_id_generator,
+    )
+    baker.make(FareRule, feed=feed, fare=fare, route=route)
+    rider_categories = baker.make(
+        RiderCategory,
+        feed=feed,
+        source_id=seq("source_id of test rider category "),
+        name=seq("name of test rider category "),
+        description=seq("description of test rider category "),
+        _quantity=3,
+    )
+    baker.make(
+        FareRiderCategory,
+        feed=feed,
+        api_id=api_id_generator,
+        fare=fare,
+        rider_category=iter(rider_categories),
+        currency_type="EUR",
+        price=seq(0),
+        _quantity=len(rider_categories),
+    )
+
+    with django_assert_max_num_queries(7):
+        endpoint = "/v1/routes/"
```

Comment:
There is also module-level `ENDPOINT`

Reply:
Ah, indeed, forgot to refactor this after rebasing.